external/aws/output.tf

```diff
@@ -23,25 +22,18 @@ resource "local_file" "ansible_inventory" {
     proxies  = { for aws_instance in aws_instance.proxy: aws_instance.tags.Name => aws_instance.private_ip },
     elk      = { (aws_instance.elk.tags.Name) = aws_instance.elk.private_ip },
     username = var.image_username,
-    key      = local_sensitive_file.ssh_private_key.filename
+    key      = var.key_name
```

Comment:
This was changed in https://github.com/redteaminfra/redteam-infra/commit/6fcec82aa22d17858f3510fc87590c6ac8d59af4#diff-def107f03f40ee472e72195dd7c71414bd1b257e8e24a48ce97817ad7ec18f76, why is this commit reverting it?